modules/services/mako.nix

```diff
@@ -4,17 +4,28 @@ with lib;
 
 let
 
-  cfg = config.programs.mako;
+  cfg = config.services.mako;
 
 in {
   meta.maintainers = [ maintainers.onny ];
 
+  imports =
+    [ (mkRenamedOptionModule [ "programs" "mako" ] [ "services" "mako" ]) ];
```

Comment:
Is this better than `mkRemovedOptionModule`?

Reply:
Yeah, that does indeed seem nicer.